src/index.mako.html

```diff
@@ -712,82 +712,66 @@
         $.ajaxSetup({
           cache: true
         });
+        function getParam(name) {
+          return decodeURIComponent((new RegExp('[?|&]' + name + '=' + '([^&;]+?)(&|#|;|$)').exec(location.search)||[,""])[1].replace(/\+/g, '%20')) || null;
+        }
 
+        var adminUrlRegexp = new RegExp('${admin_url_regexp}');
         var module = angular.module('geoadmin');
         var cacheAdd = '${version}' != '' ? '/' + '${version}' : '';
         var pathname = location.pathname.replace(/(index|mobile|embed)\.html$/g, '');
-
-        function getParam(name){
-          return decodeURIComponent((new RegExp('[?|&]' + name + '=' + '([^&;]+?)(&|#|;|$)').exec(location.search)||[,""])[1].replace(/\+/g, '%20'))||null;
-        }
-
-        var adminUrlRegexp = new RegExp('${admin_url_regexp}');
-        var wmsUrl = location.protocol + getParam('wms_url');
-        wmsUrl = adminUrlRegexp.test(wmsUrl) ? wmsUrl : location.protocol + '${wms_url}';
-        var printUrl = location.protocol + getParam('print_url');
-        printUrl = adminUrlRegexp.test(printUrl) ? printUrl : location.protocol + '${print_url}';
-
-        var apiParam = getParam('api_url');
-        var apiEnv = getParam('api_env');
-        var apiUrlBase = '${api_url}';
-
-        if (apiEnv){
-          if (apiEnv == 'prod'){
-            host = '//api3.geo.admin.ch';
-          } else {
-            host = '//mf-chsdi3.' + apiEnv + '.bgdi.ch';
+        pathname = '${s3basepath}' == '' ? pathname : '${s3basepath}';
```

Comment:
It's actually because we move index.html to the root. So pathname (dynamic) is subject to changes when we always want to point to the same resources.